File: python/1134.py
a, g, d = 0, 0, 0

while (True):
    i = int(input())

    if (i==1):
        a+=1
    elif(i==2):
        g+=1
    elif(i==3):
        d+=1
    elif(i==4):
        break
    

print(f"MUITO OBRIGADO")
print(f"Alcool: {a}")
print(f"Gasolina: {g}")
print(f"Diesel: {d}")

File: js/1134.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

let a = 0
let g = 0
let d = 0
while(1){
    i = lines.shift()

        if (i==1){
            a++;
        }else if(i==2){
            g++;
        }else if(i==3){
            d++;
        }else if(i==4){
            break;
        }

}

console.log("MUITO OBRIGADO");
console.log("Alcool: "+ a);
console.log("Gasolina: "+ g);
console.log("Diesel: "+ d);